libclamav/clamav.h

```diff
@@ -940,7 +997,23 @@ extern cl_error_t cl_scanfile_callback(const char *filename, const char **virnam
 /* ----------------------------------------------------------------------------
  * Database handling.
  */
+
+/**
+ * @brief Load the signature databases found at the path.
+ *
+ * @param path      May be a file or directory.
+ * @param engine    The engine to load the signatures into
+ * @param signo     [out] The number of signatures loaded
```

Comment:
This is a nitpick, but I think Doxygen uses `@param[out]`

Reply:
I always forget 😅  I probably make that mistake again just last night working on the OLE2 image extraction PR.  Will revisit.